addons/website_sale/models/sale_order.py

```diff
@@ -92,7 +94,12 @@ def _cart_update(self, cr, uid, ids, product_id=None, line_id=None, add_qty=0, s
             if not line_id:
                 values = self._website_product_id_change(cr, uid, ids, so.id, product_id, qty=1, context=context)
                 line_id = sol.create(cr, SUPERUSER_ID, values, context=context)
-                sol._compute_tax_id(cr, SUPERUSER_ID, [line_id], context=context)
+                try:
```

Comment:
@tivisse , It will raise Validation error from TaxCloud on Back-end and display to user , but for front-end  it will be reaised silently if we can raise validation site will be crahsed. so if validation error raised on front-end it will be failed silently on front-end like Shipping delivery module.

Thanks